src/components/Common/Breadcrumbs.tsx

```
@@ -0,0 +1,180 @@
+  if (breadcrumbs.length === 0) return null;
+
+  return (
+    <div className="flex items-center gap-4 mb-3">
+      {to ? (
+        <>
+          <div className="flex items-center gap-2">
+            <ChevronLeft className="size-4" />
+            <span
+              onClick={() => navigate(to)}
+              className="text-sm text-gray-800 hover:cursor-pointer underline font-semibold"
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)